trace-normalization/Cargo.toml

```
@@ -0,0 +1,16 @@
+[package]
+name = "datadog-trace-normalization"
+version = "0.0.0"
```

```suggestion
version = "2.0.0"
```

We are preparing to release libdatadog v2.0.0. Sound good to you?